vector/src/main/java/im/vector/app/features/home/room/detail/RoomDetailPendingAction.kt

```diff
@@ -17,6 +17,7 @@
 package im.vector.app.features.home.room.detail
 
 sealed class RoomDetailPendingAction {
+    object DoNothing : RoomDetailPendingAction()
```

Comment:
I am just wondering if this action is really needed since we will do nothing when receiving it? Does it work without any declared action?

Reply:
Yes, this is needed to finish RoomProfileActivity at this point https://github.com/vector-im/element-android/blob/develop/vector/src/main/java/im/vector/app/features/roomprofile/RoomProfileActivity.kt#L117

Reply:
I see thanks for explaining.